robotpy_installer/installer.py

```diff
@@ -1076,13 +1076,39 @@ def download_opkg_opts(self, parser):
             default=False,
             help="When upgrading, reinstall all packages even if they are already up-to-date.",
         )
+        parser.add_argument(
+            "-r",
+            "--requirement",
+            action="append",
+            default=[],
+            help="Download from the given requirements file. This option can be used multiple times.",
+        )
         parser.add_argument("--no-index", action="store_true", default=False)
 
     def install_opkg_opts(self, parser):
         self.download_opkg_opts(parser)
         parser.add_argument(
             "--ignore-image-version", action="store_true", default=False
         )
+        parser.add_argument(
+            "-r",
+            "--requirement",
+            action="append",
+            default=[],
+            help="Install from the given requirements file. This option can be used multiple times.",
+        )
+
+    def _load_opkg_from_req(*files):
+        """
+            Pull the list of opkgs from the files
+        """
+        opkgs = []
+        for file in files:
+            with open(file, "r") as f:
+                for row in f.readlines():
+                    if not row.startswith("#"):
+                        opkgs.append(row)
```

Comment:
What about trailing whitespace in the file? Also, is there a way to specify versions?

Reply:
Got this fixed. Version control doesn't seem to be a thing that's built into the installer currently or at least it isn't documented.